contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2648,6 +2648,17 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 
 							if (from_windows && orig_loginname)
 							{
+								
+								/* 
+								 * Check whether the domain name is supported 
+								 * or not
+								 */
+								char *nt_service_pointer = strcasestr(orig_loginname, "nt service");
+								if(nt_service_pointer != NULL && nt_service_pointer==orig_loginname) 
```

Comment:
Nit: Space should be there before and after == in `nt_service_pointer==orig_loginname`